pkg/agent/openflow/pipeline_windows.go

```diff
@@ -281,5 +281,17 @@ func (c *client) hostBridgeUplinkFlows(localSubnet net.IPNet, category cookie.Ca
 			Cookie(c.cookieAllocator.Request(category).Raw()).
 			Done(),
 	}
+	// The packages received from uplink are forwarded to LOCAL interface by default.
+	// If NoEncap is enabled, the reply packets from remote Pod can be forwarded to local Pod directly.
+	// by explicitly resubmitting them to EgressRuleTable and marking "macRewriteMark" at same time.
```

Comment:
I am worried about the asymmetric tx/rx paths (which might break the connection state if any stateful services - like Win firewall? - on the host is tracking connections). Is it possible to let the return traffic go br-int and host stack, and route back?

Reply:
I will try.